ddtrace/tracer/textmap.go

```diff
@@ -705,12 +708,17 @@ func composeTracestate(ctx *spanContext, priority int, oldState string) string {
 		if !strings.HasPrefix(k, "_dd.p.") {
 			continue
 		}
+		kWithSub := keyRgx.ReplaceAllString(k[len("_dd.p."):], "_")
+		vWithSub := valueRgx.ReplaceAllString(v, "_")
+		if kWithSub != k || vWithSub != v {
+			ctx.trace.setTag(keyPropagationErrorW3C, "invalid_tag_characters")
```

Comment:
From offline: this tag is used in testing. But apart from testing, we will be increasing the size of the span with the tag that’s not used anywhere, which is probably wahat we want to avoid?